packages/react/src/components/Chip/_ChipButton/ChipButton.tsx

```diff
@@ -0,0 +1,49 @@
+import type { HTMLAttributes } from 'react';
```

Comment:
Created this "base" component to avoid duplicating CSS and sizing logic. This is used as the Base for the other Chips.